pywr/solvers/__init__.py

```diff
@@ -95,6 +99,12 @@ def __init__(self, *args, **kwargs):
                 "PYWR_SOLVER_GLPK_FIXED_FACTORS_ONCE",
                 bool,
             )
+            kwargs = _parse_env_kwarg(
+                kwargs,
+                "use_safe_api",
+                "PYWR_SOLVER_GLPK_UNSAFE_API",
```

Comment:
This is a little confusing. The env variable is for the unsafe api which is setting an arg for the safe api. Does one of them need updating?

Reply:
This is actually incorrect. Well spotted.